### PR TITLE
Add instructions to enable Infoblox integration

### LINKED ISSUE #520: Infoblox Installation Missing Enable
### Environment
* Python version:  3.8+
* Nautobot version:  2.3.0
* nautobot-ssot version:  3.0.0

### Expected Behavior
Expected to see full instructions on installing and setting up Infoblox integration.

### Observed Behavior
Found the documentation missing a note about setting `enable_infoblox` to True in PLUGINS_CONFIG for `nautobot_ssot`. It appears this was accidentally removed during the documentation revamp when the settings were moved to a config object.

### Steps to Reproduce
1. Review Infoblox integration installation instructions.
2. Attempt to follow and find Infoblox integration is not enabled.
3.

### PR DESCRIPTION
**Pull request by nopg**
Documentation updated, Closes issue #520

### CODE REVIEW

File: docs/admin/integrations/infoblox_setup.md [hunk 1]
```diff
@@ -15,7 +15,19 @@ pip install nautobot-ssot[infoblox]
 !!! note
     Legacy configuration settings defined in the `nautobot_config.py` and environmental variables are deprecated. These settings are migrated on a best-effort basis on the first startup following migration to the Nautobot SSOT 2.7.0 or higher.
 
-Integration configuration is defined in the instance of the `SSOTInfobloxConfig` model. Multiple configuration instances are supported. Synchronization jobs take the `Config` parameter which specifies the configuration instance to use.
+The integration with Infoblox utilizes [External Integrations](https://docs.nautobot.com/projects/core/en/stable/user-guide/platform-functionality/externalintegration/?h=external) to specify your Infoblox host information. To enable this integration, the only modification needed is to activate it in the nautobot_config.py file.
+
+Below is an example snippet from `nautobot_config.py` that demonstrates how to enable the Itential integration:
```
Comment (jdrew82) [inline docs/admin/integrations/infoblox_setup.md:20]: This should be Infoblox not Itential.

**Review by jdrew82 — CHANGES_REQUESTED** [review]
Just a small correction needed but looks good to me.

File: docs/admin/integrations/infoblox_setup.md (hunk 1 above)
Comment (nopg) [reply docs/admin/integrations/infoblox_setup.md:20]: Oops!

**Review by jdrew82 — APPROVED** [review]
LGTM!

**Pull request merged**